demos/basics/loading-pandas.ipynb

```diff
@@ -0,0 +1,3961 @@
+{
```

Comment:
<p>Given that we established <code>subject</code>  as being the true label we want to use for ML tasks, I wonder if showing edge weights being derived from these true labels is not the best example to be showing..? Maybe just using random numbers is fine?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1074/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
I'm not sure. I kinda wanted to provide a realistic example of actually computing edge weights, but maybe it's fine to just use random numbers as a proxy for a column that already exists in the edge data.

Reply:
I've switched to using completely synthetic data per Tim's comment below, so I guess this is invalidated. (Although I had also switched to using 1, 2, ..., 5429 as the weights for the 5249 edges.)